src/dream.ml

```diff
@@ -18,10 +18,11 @@ let next ~buffer ~close ~exn request =
   next ~buffer ~close ~exn request
 
 include Dream__middleware.Log
-include Dream__middleware.Echo
+include Dream__middleware.Log.Make (Ptime_clock)
+
+let () = initialize ()
```

Comment:
This should probably be qualified with a module path or at least commented, because right now, it is unclear that this function has something to do with logging, or where to look it up.

Reply:
I added a comment when the function comes from the result of the functor above. We can rename it if you want.